streams/pm.m3u

```diff
@@ -0,0 +1,3 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="SaintPierreMiquelon1ere.pm" status="online",Saint-Pierre & Miquelon 1ere [Geo-blocked]
+https://la1ere.francetvinfo.fr/saintpierremiquelon/
```

Comment:
https://live-outremer.ftven.fr/372a0c4d-b466-4c86-9e40-73ee3e1b7073_0_HLS-m/manifest.m3u8

token with a simple GET to https://hdfauth.ftven.fr/esi/TA?format=json&url=https%3A%2F%2Flive-outremer.ftven.fr%2F372a0c4d-b466-4c86-9e40-73ee3e1b7073_0_HLS-m%2Fmanifest.m3u8

Reply:
Again, edit as you like: the ball is now rolling ;)